src/clojure_tictactoe/tictactoe.clj

```diff
@@ -1,31 +1,78 @@
 (ns clojure-tictactoe.tictactoe
-  (:gen-class))
+  (:gen-class)
+  (:require [clojure.string :as string]))
 
 
-(def welcome-message 
+(def welcome-message
   "Welcome to the game of TicTacToe! \n")
 
-(def instructions-message 
+(def instructions-message
  "Instructions: 2 players take turns placing markers, try to get three of your markers in a row to win! \n
 
-Examples: 
+Examples:
 
   Vertical      Horizontal      Diagonal
 
-   | x |     |    |   |     |  x |   |     
-===+===+===  | ===+===+===  | ===+===+===  
-   | x |     |  x | x | x   |    | x |     
-===+===+===  | ===+===+===  | ===+===+===  
+   | x |     |    |   |     |  x |   |
+===+===+===  | ===+===+===  | ===+===+===
+   | x |     |  x | x | x   |    | x |
+===+===+===  | ===+===+===  | ===+===+===
    | x |     |    |   |     |    |   | x  \n\n ")
 
 
 (defn print-game-intro []
   (println welcome-message)
   (println instructions-message))
 
+;TODO make this DRYer when you have minute and a better understanding of Clojure
 (defn board [spaces]
-  " 0 | 1 | 2 \n===+===+===\n 3 | 4 | 5 \n===+===+===\n 6 | 7 | 8 \n")
+ (str  " " (spaces 0) " | " (spaces 1) " | " (spaces 2) " \n===+===+===\n " (spaces 3) " | " (spaces 4) " | " (spaces 5) " \n===+===+===\n " (spaces 6) " | " (spaces 7) " | " (spaces 8) " \n"))
+
+(defn get-user-input []
+  (let [user-input (string/trim (read-line))]
+    (if (= user-input "quit")
+      (. System exit 0)
+      user-input)))
 
 (defn continue-to-game []
   (println "Press enter key to continue")
-  (read-line))
+  (get-user-input)
+  nil)
```

Comment:
Is this `nil` necessary?

Reply:
Dunno about strictly "necessary" but it returns an empty string otherwise and I'd prefer the nil. It seems like a more logical response for a function thats focus isn't on what it returns.